calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
 title: View flow logs
 ---
 
+import Screenshot from '../../src/___new___/components/Screenshot';
+
 # View flow logs in the Calico Whisker web console
+
+You can observe network traffic in your cluster by viewing flow logs in the Calico Whisker web console.
+
+## About flow logs in Calico Open Source
+
+Tracking network activity in your cluster is essential for understanding how your applications communicate.
+Calico helps you see your network activity by processing your network flows and recording them in a digestible format called a flow log.
+
+A flow log isn't a record of every single network connection that takes place in your cluster.
+Rather, it is an aggregated summary of a type of network flow that occurs in your cluster over a period of time.
+Imagine, for example, that your cluster has 1000 individual network flows that take place within 1 minute.
+It would be difficult to sort through those 1000 flows, and more difficult to understand what they mean.
+But the corresponding flow logs for that minute might turn those 1000 entries into 15 flow logs, each representing a distinct type of flow (Pod A to Pod B, public internet to pod C), but without the duplicate flow type entries.
+This way, you can actually understand the overall network activity without being overwhelmed by the sheer volume of data.
+
+### Using flow logs
+
+Flow logs are useful for a variety of purposes, including:
+* **Network visibility.**
+  Understanding traffic patterns within the cluster, between namespaces, and to or from external services.
+* **Troubleshooting connectivity issues.**
+  If pods aren't communicating, flow logs can show if traffic is being denied by policy, if it's reaching the destination
+* **Security.**
+  Detecting policy violations (unexpected deny logs), identifying anomalous traffic patterns (e.g., unusually high data transfer), and investigating potential security incidents.
+* **Policy validation.**
+  Confirming that network policies are working as intended (allowing expected traffic, denying forbidden traffic).
+  This is especially useful for testing out [staged network policies](../network-policy/staged-network-policies.mdx) before moving them to production.
+
+### Calico Whisker
+
+The Calico Whisker web console is a browser-based interface that shows you a stream of flow logs from your cluster in real time.
```

Comment:
"real time" is up for debate - the flows show up in the console after some delay as we need to aggregate data from multiple sources before displaying it. Maybe "near real time" ?

Reply:
I'd say it's true to the spirit of 'real time', even if it's pulling in data every 15 seconds. You could argue the same no matter what the interval, even if it was 15 ms.

But if you think this will mislead or confuse readers, let me know. It seems okay to me.